job_runner/reporting/event.py

```diff
@@ -87,78 +98,58 @@ def _init_handler(self, msg):
         self._step_id = msg.step_id
         self._event_publisher_thread.start()
 
-    def _job_handler(self, msg):
-        job_path = f"{self._step_path()}/job/{msg.job.index}"
-
+    def _job_handler(self, msg: Message):
+        job_name = msg.job.name()
+        logger.debug(f"Job: {job_name} is being handle")
+        job_data = {
+            "source": f"{self._step_path()}/job/{msg.job.index}",
+            "datacontenttype": "application/json",
+        }
         if isinstance(msg, Start):
-            self._dump_event(
-                CloudEvent(
-                    {
-                        "type": _FM_JOB_START,
-                        "source": job_path,
-                        "datacontenttype": "application/json",
-                    },
-                    {
-                        "stdout": str(Path(msg.job.std_out).resolve()),
-                        "stderr": str(Path(msg.job.std_err).resolve()),
-                    },
-                )
+            logger.debug(f"{job_name} job is starting")
+            self._dump_cloud_event(
+                attributes={"type": _FM_JOB_START, **job_data},
+                data={
+                    "stdout": str(Path(msg.job.std_out).resolve()),
+                    "stderr": str(Path(msg.job.std_err).resolve()),
+                },
             )
             if not msg.success():
-                self._dump_event(
-                    CloudEvent(
-                        {
-                            "type": _FM_JOB_FAILURE,
-                            "source": job_path,
-                            "datacontenttype": "application/json",
-                        },
-                        {
-                            "error_msg": msg.error_message,
-                        },
-                    )
+                logger.warning(f"{job_name} job failed to start successfully")
```

Comment:
Is warning right here? The job failed, it's not a warning about anything. Either use `debug` or `error`.

https://dave.cheney.net/2015/11/05/lets-talk-about-logging

> Eliminate the warning level, it’s either an informational message, or an error condition.

Reply:
agree